metabase/metabase
# [FE] Allow multiple breakouts for a column

Epic https://github.com/metabase/metabase/issues/46536

How to verify:
- New -> Question -> Orders
- Summarize -> Count
- Breakout -> Created At (Year) 
- Breakout -> Created At (Quarter of Year) 
- Visualize
- Summarize (button) -> You should be able to change temporal bucketing for individual columns
- Filter (button) -> You should be able to apply filters to individual columns

Please note that names of the columns can be duplicate sometimes. This is not great but we can improve it later **as long as the column itself works correctly**.
